lib/propolis/src/hw/keyboard/mod.rs

```diff
@@ -0,0 +1,321 @@
+mod scan_code_1;
+mod scan_code_2;
+
+use std::convert::TryFrom;
+
+use anyhow::{anyhow, Result};
+use ascii::AsciiChar;
+use rfb::keysym::Keysym::{self, *};
+use rfb::rfb::KeyEvent;
+
+use super::ps2ctrl::PS2ScanCodeSet;
+use scan_code_1::*;
+use scan_code_2::*;
```

Comment:
The scancodes are PS/2 specific, right?  Perhaps it makes sense for those modules to live under ps2?

Reply:
sure, the structure you suggested makes sense to me. (I don't think there's much that could be shared with another keyboard here, since it's mostly about scan code translation.)

Reply:
done in 587c21e